src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String STRING_START_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String lineSeparator = System.lineSeparator();
+            String lineFromFile = bufferedReader.readLine();
+            while (lineFromFile != null) {
+                stringBuilder.append(lineFromFile).append(lineSeparator);
+                lineFromFile = bufferedReader.readLine();
+            }
+            String[] splitedStringBuilderResult = stringBuilder
```

Comment:
move the following code out of the `try` block
in case of compilation problems, move the declaration of the needed variables out of the `try` block as well

Reply:
Sorry but I didn't know git hub ruls - and I thought that I should move code from 20 line. Hopefully mentor explane what your comment connets to line 23
I moved code